### DISCUSSION TITLE
Is my domain still active?

### DISCUSSION THREAD

**Discussion by bi0hazarDD** [is-a-dev/register#6945] [Support] [answered]
I wanted to know if my domain, jonathancarrillo is still active. A few days ago I had to verify my gh pages challenges, which I am still waiting for my pr to be approved for. However, I'm a bit lost as if my domain was still active or was unverified, meaning I have to register it once again. 

Could someone give some insight into this please? 

Many thanks!

**Comment by wdhdev** [comment, marked as answer]
Please read this issue: #6627
